src/components/modal/modal.styles.js

```diff
@@ -1,32 +1,84 @@
 import { makeStyles } from '@material-ui/core/styles';
 
 export const useStyles = makeStyles((theme) => ({
+  message: {
+    textAlign: 'center',
+    fontWeight: 'bold',
+    marginBottom: 5,
+    position: 'relative',
+    marginTop: '-15px'
+    // fontSize:'5px'
+  },
+  // '@media (max-width: 950px)': {
+  //   message: {
+  //     marginTop: 3,
+  //     marginLeft: '5px',
+  //     position: 'absolute',
+  //     top: '5px',
+  //     float: 'left'
+  //   }
+  // },
   paper: {
     position: 'absolute',
-    top: '50%',
-    left: '50%',
+    top: '53%',
+    left: '51%',
+    // marginLeft:'50%',
+    // marginTop:'210px',
     transform: 'translate(-50%, -50%)',
+    height: '30%',
     width: 400,
-    backgroundColor: theme.palette.background.paper,
-    border: '2px solid #000',
+    backgroundColor: '#F1F3F4',
+    color: 'black',
+    border: '5px solid #666',
     boxShadow: theme.shadows[5],
-    padding: theme.spacing(2, 4, 3),
-    '& p': {
-      fontSize: '1.2em'
-    }
+    padding: theme.spacing(2, 4, 3)
+
+    //   '@media (max-width: 950px)': {
+    //     width: '320px',
+    //     height: '8%',
+    //     top: '69%',
+    //     left: '38%',
+    //     '& p': {
+    //       fontSize: '1em'
+    //     }
+    //   }
   },
   buttonGroup: {
     width: '100%',
+    height: '30%',
     textAlign: 'center',
     '& > button': {
       color: theme.palette.button.normal.color,
       backgroundColor: theme.palette.button.normal.backgroundColor,
-      padding: 5,
-      margin: 5,
+      paddingBottom: '5px',
+      margin: 3,
+      width: '130px',
       '&:hover': {
         color: theme.palette.button.hover.color,
         backgroundColor: theme.palette.button.hover.backgroundColor
       }
     }
   }
+  // '@media (max-width: 950px)': {
```

Comment:
Why you commented this lines?

Reply:
I guess I have just forgot to delete this commented code.Now it is deleted